app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -40,7 +41,18 @@
       android:text="@={viewModel.answerText}"
       android:textColor="@color/oppiaPrimaryText"
       android:textColorHint="@color/edit_text_hint"
-      android:textSize="16sp"
-      android:textStyle="italic" />
-  </FrameLayout>
+      android:textSize="14sp"
+      android:textStyle="italic"
+      app:textChangedListener="@{viewModel.passwordTextWatcher}" />
```

Comment:
Nice--this is a really clean way to solve the automatic resetting.

Reply:
thank you